Makefile

```diff
@@ -42,9 +44,18 @@ build-sdk-python2: .GOPATH/.ok
 	$Q go build $(if $V,-v) $(VERSION_FLAGS) -buildmode=c-shared -o bin/python2/ubbagent.so $(IMPORT_PATH)/sdk/python2
 	@rm bin/python2/ubbagent.h
 
+.PHONY: build-sdk-python3
+build-sdk-python3: .GOPATH/.ok
+	$Q go build $(if $V,-v) $(VERSION_FLAGS) -buildmode=c-shared -o bin/python3/ubbagent.so $(IMPORT_PATH)/sdk/python3
+	@rm bin/python3/ubbagent.h
+
 .PHONY: test-sdk-python2
 test-sdk-python2: build-sdk-python2
-	$Q PYTHONPATH=bin/python2 python sdk/python2/test.py
+	$Q PYTHONPATH=bin/python2 python2 sdk/python2/test.py
```

Comment:
I can't tell, but if the files in both directories are identical, it's much better to avoid duplicate code. We can rename it to `sdk/python` and use it here.

Reply:
They are not identical. Modifications had to be done to make it work with py3.